website/markdown/posts/2020-06-17-version-1.11.0/post.mdx

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: Easily setup signing for your project with Tuist 1.11.0 Volare
+categories:
+  [
+    Tuist,
+    release,
+    swift,
+    project generation,
+    xcode,
+    apple,
+    '1.11.0',
+    'fastlane',
+    'automation',
+  ]
+excerpt: The new Tuist release 1.11.0 Volare introduces a signing feature to help you with maintaining and integrating your signing artifacts.
+author: fortmarek
+---
+
+Hi! 👋
+
+I am really excited to write this blog post as we have a great new feature for you to use -
+and this feature is signing 🔑
+
+So, let's dive right into it 🏊‍♂️
+
+## Signing
+
+Signing is one of the most complicated domains that iOS developers 
+have to deal with in Xcode. And although Apple has introduced automatic signing feature,
+most teams have decided not to use it as it creates a messy environment for their team.
+
+While for automating the process of generating provisioning profiles you'll have to wait as
+that is still work in progress, in this release we bring you easy integration of already
+created provisioning profiles and certificates.
+
+With the new feature, you'll be able to put your provisioning profiles and certificates 
+into `Tuist/Signing` directory and ... just run `tuist generate`. That's it!
+No figuring out how to set up the Xcode build settings as that is done for you.
+And as a plus your signing artifacts will be automatically encrypted with
+your key at `master.key`, so you can safely push it to your remote
+repository, so other team members can run the apps on a device, too ✨
+You can find more info in the [documentation](https://tuist.io/docs/signing/signing/)
+
+You can also check out an introduction [video](https://youtu.be/WGKp1EHcpME) where we
```

Comment:
I'd embed the video instead of adding a link to it. If I'm not wrong, I think you can just paste the link and one of the Gatsby plugins that we are using will turn it into embedded content.

Reply:
Huh, did not know about that, thanks for the suggestion 👍